Index: ezsdk-5_03_00_09/component-sources/omx_05_02_00_26/include/OMX_TI_Video.h
===================================================================
--- ezsdk-5_03_00_09.orig/component-sources/omx_05_02_00_26/include/OMX_TI_Video.h	2011-12-23 09:35:05.461812640 -0600
+++ ezsdk-5_03_00_09/component-sources/omx_05_02_00_26/include/OMX_TI_Video.h	2011-12-23 09:35:23.999221898 -0600
@@ -57,6 +57,8 @@
 #include "OMX_Core.h"
 
 #ifdef _VIDEO_M3_DYNAMIC_CONFIG
+#include <xdc/std.h>
+
 #include <ti/sdo/ce/video2/videnc2.h>
 #include <ti/sdo/ce/video3/viddec3.h>
 #ifdef CODEC_H264ENC
Index: ezsdk-5_03_00_09/component-sources/omx_05_02_00_26/src/ti/omx/interfaces/openMaxv11/OMX_TI_Video.h
===================================================================
--- ezsdk-5_03_00_09.orig/component-sources/omx_05_02_00_26/src/ti/omx/interfaces/openMaxv11/OMX_TI_Video.h	2011-12-23 09:35:58.649975633 -0600
+++ ezsdk-5_03_00_09/component-sources/omx_05_02_00_26/src/ti/omx/interfaces/openMaxv11/OMX_TI_Video.h	2011-12-23 09:36:26.426238494 -0600
@@ -57,6 +57,8 @@
 #include "OMX_Core.h"
 
 #ifdef _VIDEO_M3_DYNAMIC_CONFIG
+#include <xdc/std.h>
+
 #include <ti/sdo/ce/video2/videnc2.h>
 #include <ti/sdo/ce/video3/viddec3.h>
 #ifdef CODEC_H264ENC
